### PR TITLE
add mztab2tsv

### PR DESCRIPTION
**Pull request by bernt-matthias**
using pyteomics

### CODE REVIEW

**Comment by bernt-matthias** [comment]
python script by @annalefarova

File: tools/pyteomics/mztab2tsv.xml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<tool id="mztab2tsv" name="pyteomics: convert mztab to tabular" version="@TOOL_VERSION@" profile="20.01" license="TODO_license_in_spdx_format">
```
Comment (bgruening) [inline tools/pyteomics/mztab2tsv.xml:1]: maybe the name just "pyteomics" and the rest as description?

license is missing ... its the license of the python script

File: tools/pyteomics/mztab2tsv.xml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    <edam_operations>
+        <edam_operation>operation_3434</edam_operation><!--	Convert a data set from one form to another -->
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">pyteomics</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+    ## make sure that selected outputs exist even if the tool does not generate them
+    #for o in $out_select
+        touch "$o".tsv &&
```
Comment (bgruening) [inline tools/pyteomics/mztab2tsv.xml:27]: single quotes

File: tools/pyteomics/mztab2tsv.xml [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            <option value="smf">Small molecule feature table (v2)</option>
+            <option value="sme">Small molecule evidence table (v2)</option>
+        </param>
+    </inputs>
+    <outputs>
+        <expand macro="output" type="mtd" label="Metadata"/>
+        <expand macro="output" type="prt" label="Proteins"/>
+        <expand macro="output" type="pep" label="Peptides"/>
+        <expand macro="output" type="psm" label="Peptide spectrum matches"/>
+        <expand macro="output" type="sml" label="Small molecules"/>
```
Comment (bgruening) [inline tools/pyteomics/mztab2tsv.xml:48]: are all filetypes available in Galaxy?

**Review by bgruening — COMMENTED** [review]
Very nice, only some small suggestions.

**Comment by bgruening** [comment]
@annalefarova that looks great! Thanks a lot.

File: tools/pyteomics/mztab2tsv.xml (hunk 3 above)
Comment (bernt-matthias) [reply tools/pyteomics/mztab2tsv.xml:48]: these are all tabular files .. see the macro?

File: tools/pyteomics/mztab2tsv.xml (hunk 2 above)
Comment (bernt-matthias) [reply tools/pyteomics/mztab2tsv.xml]: done

File: tools/pyteomics/mztab2tsv.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/pyteomics/mztab2tsv.xml:1]: preferences anyone .. ? The license question is always a bit annoying to me .. I would like https://en.wikipedia.org/wiki/WTFPL :)

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Merci! @annalefarova do you want to do step 3 as well :) https://galaxyproject.eu/posts/2020/08/22/three-steps-to-galaxify-your-tool/

**Comment by annalefarova** [comment]
@bgruening thank you, I can try to do that :)
Besides I've also updated my function to write an empty `tsv` file instead of file with an empty string when some table is missing (@bernt-matthias asked about that). Can I create PR with an update to [this file](https://github.com/bernt-matthias/tools-galaxyp/blob/cef579d20013d6b67ac4f682191276fe426c78bc/tools/pyteomics/mztab_reader.py).  Only internal implementation has changed the interface remained the same.

**Comment by bgruening** [comment]
Yes, this would be the way. Just PR this file and tests will run.

**Comment by bernt-matthias** [comment]
> For step 3, which tool_panel_section_label should I write? 

Metabolomics or proteomics? I guess Massspec.. does not exist. 

> And the name is read_mztab, right?

Should be mztab2tsv

**Pull request merged**